### PR TITLE
Simple HTML-enabling fix

### LINKED ISSUE #359: Link not clickable in confirm email message sent
email blech

Also guessed @MisterJames is likely one to fix given he worked on emails

### PR DESCRIPTION
**Pull request by MisterJames**
This is not the be-all-end-all solution. We need to progress (longer term) the templates idea from SendGrid.

- extended models to add HTML field
- updated view models affected by HTML
- updated email confirmation to set HTML
- updated service to push HTML out with email

Fixes #359

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady/Services/MessageServices.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public Task SendEmailAsync(string email, string subject, string message)
                 {
                     EmailMessage = message,
                     EmailRecipients = new List<string> {email},
+                    HtmlMessage = message,
```
Comment (MisterJames) [inline AllReadyApp/Web-App/AllReady/Services/MessageServices.cs]: @tonysurma This is the one that addresses the HTML aspect, the link not rendering in the email.

File: AllReadyApp/NotificationsProcessor/Functions.cs [hunk 2]
```diff
@@ -47,6 +47,7 @@ public static void ProcessEmailQueueMessage([QueueTrigger("email-pending-deliver
             myMessage.AddTo(emailMessage.Recipient);
             myMessage.From = new MailAddress(from, "AllReady");
             myMessage.Subject = emailMessage.Subject;
+            myMessage.Html = emailMessage.HtmlMessage;
```
Comment (MisterJames) [inline AllReadyApp/NotificationsProcessor/Functions.cs]: @tonysurma This is what is passed to SendGrid. We **really** want to think about refining these messages.

**Comment by MisterJames** [comment]
@dpaquette and @tonysurma: End-to-end confirmed:

image

 - Message is queued
 - HTML is set correctly (although the message should be prettied up)
 - Message arrives as HTML and is clickable
 - Controller picks up and processes confirmation as sent to user

image

**Comment by dpaquette** [comment]
This looks good. As @MisterJames mentioned, we do need to refine the actual messages we are sending but at least this gives us the structure we need to send both Html and Plain Text emails.

:shipit:

**Comment by tonysurma** [comment]
Excellent, thanks

**Pull request merged**